app/src/views/Login.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { StyleSheet, Text, View, Image, Button } from "react-native"
+// import * as Expo from "expo"
+import Main from './Main';
+import * as Google from "expo-google-app-auth";
+import axios from 'axios';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      signedIn: false,
+      name: "",
+      photoUrl: ""
+    }
+  }
+  signIn = async () => {
+    try {
+      const { type, accessToken, user} = await Google.logInAsync({
+        androidClientId:
+          "847385043058-1v26bjo0eahcc7sal4f86s7ed04mtjro.apps.googleusercontent.com",
+        iosClientId: '847385043058-lo98oseuebn051god5a24pacgevrubp0.apps.googleusercontent.com',
+        scopes: ["profile", "email"]
+      })
+    
+      if (type === "success") {
+        console.log('google user',user);
+        this.setState({
+          signedIn: true,
+          name: user.name,
+           photoUrl: user.photoUrl
+        })
+        return axios.post('http://localhost:8080/profile/user/', {
+           user,
+          })
+            .then(function (response) {
+              // get the user from the database 
```

Comment:
Whats happening in here?

Reply:
I was going to get the user info from the data base and post it to the page